plugins/ingest/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
     // we don't use Maxmind's http service:
     exclude group: 'com.google.http-client', module: 'google-http-client'
   }
+  compile 'joda-time:joda-time:2.8.2'
```

Comment:
I initially thought that this would be a problem, but a plugin can have its version of a dependency thanks to the fact that each plugins runs isolated in its own class loader.